e2e-tests/IIoTPlatform-E2E-Tests/TestHelper.cs

```diff
@@ -683,5 +789,58 @@ private static PrivateKeyFile GetPrivateSshKey(IIoTPlatformTestContext context)
                 exception = exception.InnerException;
             }
         }
+
+        private static async Task<(List<(string NodeId, string NodeClass, bool Children)> results, string continuationToken)> Twin_GetBrowseEndpoint_Internal(
```

Comment:
Please add `Async` suffix to the method name. Please add a `CancellationToken` parameter and pass it down to downstream `async` calls.